src/parcel.F

```diff
@@ -1153,48 +1146,34 @@ subroutine parcel_interp(dt,mtime,xh,uh,ruh,xf,uf,yh,vh,rvh,yf,vf, &
 
       !print *,'parcel_interp: point #12'
     IF( prqsl.ge.1 )THEN
-#ifdef _B4B02F
```

Comment:
Why are we removing B4B regions in this PR?

Reply:
See the previous reply.